lld/COFF/Driver.cpp

```diff
@@ -2601,7 +2601,9 @@ void LinkerDriver::linkerMain(ArrayRef<const char *> argsArr) {
     createECExportThunks();
 
   // Resolve remaining undefined symbols and warn about imported locals.
-  ctx.symtab.resolveRemainingUndefines();
+  while (ctx.symtab.resolveRemainingUndefines()) {
+    run();
+  }
```

Comment:
I think we stylistically tend to omit the braces for these kinds of single-line statements, see https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements.